.github/workflows/ci.yml

```diff
@@ -477,14 +477,11 @@ jobs:
           toolchain: ${{ env.RUST_VERSION }}
       - uses: Swatinem/rust-cache@v2
 
-      - name: Install cargo-deps
-        run: cargo install cargo-deps
-
       # We require this check to avoid cyclic dependencies between 'fuels' and 'forc-pkg'.
       # Detailed explanation is found in the echo below.
       - name: Check 'forc-pkg' dependencies for 'fuels' crates
         run: |
-          deps=$(cargo deps --manifest-path forc-pkg/Cargo.toml)
+          deps=$(cargo tree --manifest-path forc-pkg/Cargo.toml)
```

Comment:
Good catch, it looks like cargo-deps does not support workspaces so this wasn't even running.

Reply:
Thanks for that suggestion. It worked as planned:

https://github.com/FuelLabs/sway/pull/6515#issuecomment-2337406478